tests/libtest/test_cluster_utils.py

```diff
@@ -105,6 +105,7 @@ def teardown(self):
 
 
 @tier1
```

Comment:
Is it also real tier1 test with valid polarion ID? If not, please remove also tier1 marker.

Reply:
Done